README.rst

```diff
@@ -17,151 +17,18 @@ Developed by Kitware, Inc. with funding from The National Cancer Institute of th
     :target: https://raw.githubusercontent.com/DigitalSlideArchive/DSA-WSI-DeID/master/LICENSE
     :alt: License
 
-Installation
-============
 
-Prerequisites
--------------
 
-At a minimum, you will need `Docker <https://docs.docker.com/install/>`_ and `docker-compose <https://docs.docker.com/compose/install/>`_.  You also will need a copy of this repository, either obtained via ``git`` or downloaded directly.  If you have ``git`` installed, this can be::
+Navigating the Documentation
+============================
 
-    git clone https://github.com/DigitalSlideArchive/DSA-WSI-DeID.git
 
-Install commands need to be run from the ``devops/wsi_deid`` directory.  Examples are given via a command prompt, but a desktop version of Docker will work as well.
+* docs/INSTALL.rst contains system installation and administration information, mostly focused on running terminal commands.
```

Comment:
I recommend making this actual links (e.g., `` `docs/INSTALL.rst <docs/INSTALL.rst>`_``).

Reply:
Also, the links on in `homepage.md` will have to be updated to the new locations.